index.js

```diff
@@ -25,10 +25,10 @@ BrowserSyncPlugin.prototype.apply = function (compiler) {
 
   compiler.plugin('done', function (stats) {
     if (stats.hasErrors() || stats.hasWarnings()) {
+      var error = stats.toString('minimal')
       return self.browserSync.sockets.emit('fullscreen:message', {
         title: 'Webpack Error:',
-        body: stripAnsi(stats.toString()),
-        timeout: 100000
+        body: stripAnsi(error),
       })
     }
     self.browserSync.reload()
```

Comment:
IMO, it's better to like this : 
```js
if (stats.hasErrors() || stats.hasWarnings()) {
  // ...
} else {
  self.browserSync.reload()
}
```
With this options, if we add an option `{reload: false)`, it prevent reloading page on error.
Reload is too fastest to see the proper error message.

I also like to have an option for timeout, default timeout is too short in my opinion.